### ISSUE TITLE
Accuracy not the same for the pre-trained models

### ISSUE THREAD

**Issue by SikanderBinMukaram** [XingwXiong/Face3D-Pytorch#2]
I used the pre-trained models for evaluation by correctly specifying the value of pretrained_model_path while running the scripts but couldn't get the accuracy claimed. I used the same data set and the same scripts provided but still can't get any accuracy at all. I used around 280 classes out of 1200 classes of the 3dvggface2 dataset. Also, I want to know how face depth maps for the dataset were obtained. Please let me know if what I possibly am doing wrong.

**Comment by yuanliangxie** [comment]
您好，您还有3dvggface2这个3d人脸数据集吗？
